README.md

```diff
@@ -1,6 +1,9 @@
 ## coreJSON Library
 
-A parser strictly enforcing the ECMA-404 JSON standard, suitable for microcontrollers
+This repository contains the coreJSON library, a parser that follows the ECMA-404 JSON standard and is suitable for low memory footprint embedded devices. The coreJSON library is distributed under the [MIT Open Source License](LICENSE).
+
+
+This library has gone through code quality checks including for [GNU Complexity](https://www.gnu.org/software/complexity/manual/complexity.html), [MISRA coding standard](https://www.misra.org.uk/MISRAHome/MISRAC2012/tabid/196/Default.aspx), [Coverity statical analysis](https://scan.coverity.com/), and [AWS CBMC automated reasoning tool](https://www.youtube.com/watch?v=YwQHAPRhQkI&feature=youtu.be&t=1721) to validate memory safety, thread safety and functional correctness proof.  Deviations to the MISRA C:2012 guidelines are documented under [MISRA Deviations](MISRA.md). 
```

Comment:
Some grammar issues here, as well as conflating tools used with what is being measured.  Maybe something like:

"This library is reported free from errors using both static analysis (Coverity) and formal verification (CBMC).  The library maintains a low complexity score throughout (reported by GNU Complexity).  The library complies with all mandatory MISRA 2012 rules and almost all non-mandatory rules.  Deviations..."

Reply:
worked with Tanmoy and have updated the language